### DISCUSSION TITLE
NexoPOS 5.0.2

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#1737] [Releases]
We're releasing this patch update to fix newly discovered issues. Here is the changelog.

Changelog:
========
- Added: Ensure set action perform a correct increase/decrease action
- Updated: set is the default adjustment action by default
- Removed: unused js files
- Fixed: Field was disabled when the reset confirmation was closed
- [Update customer name fields in OrdersService and invoice settings](https://github.com/Blair2004/NexoPOS/pull/1736/commits/43d8e98273c9661f9aa43885cde962af10620779)
- Fixed: scroll on product grid
- Added: Kmer Language

Special Thanks:
============
- @mvannakpanha

<hr /><em>This discussion was created from the release <a href='https://github.com/Blair2004/NexoPOS/releases/tag/v5.0.2'>NexoPOS 5.0.2</a>.</em>

**Comment by FlutterRonin** [comment]
Thank @mvannakpanha for this contribute. Also thank to NexoPOS for perfect system, but "Khmer" not "kmer". Thanks

**Reply by Blair2004** [reply]
Thank you for this. We'll update accordingly.
